static2/src/pages/text/text.js

```diff
@@ -0,0 +1,67 @@
+import React , { useState } from 'react';
+import api from "../../services/api";
+
+
+function Text () {
+
+    const [text, setText] = useState("");
+    const [lang, setLang] = useState("");
+
+   function handleNewTranslation(e) {
+      e.preventDefault();
+
+      try {
+        api.get('/text', {
+          headers: {
+            text: text,
+            lang: lang
+          }
+        }).then(response => {
+         const audio_file = response.data;
```

Comment:
Looks like a duplicate of the code we have in `Hello` component, are you planning to import this component, or it's just a leftover ?